sdk/devcenter/Azure.Developer.DevCenter/api/Azure.Developer.DevCenter.netstandard2.0.cs

```diff
@@ -1,108 +1,108 @@
 namespace Azure.Developer.DevCenter
 {
+    public partial class DeploymentEnvironmentsClient
```

Comment:
Has the API been reviewed by the architects? Just an FYI - you will need an architect review before you can release the GA SDK.

Reply:
Yes, we had an arch board review for this version of the API.